blog/2017-12-11-end-to-end-testing-kubernetes-openshift.textile

```diff
@@ -0,0 +1,263 @@
+---
+layout: post
+title: End-To-End Integration Testing For Kubernetes and OpenShift
+author: hemanik
+tags: [ kubernetes, k8s, openshift, testing, guide, cube, fabric8-maven-plugin, examples ]
+---
+*Does the title leave you wondering, why you need integration tests for your deployments ?*
+
+If so, let me give you a few examples. Consider you decide to upgrade Kubernetes, deploy a new Zapier
+integration or add some authentication capability. In order to ensure your platform works every single
+time, all of the above scenarios need to be tested with robust integration tests.
+
+With the advent of microservices and cloud native applications, adding integration or
+end-to-end tests to your Kubernetes/OpenShift projects might be challenging and cumbersome especially when 
+you want to set up the test infrastructure to be as close as possible to production.
+
+Arquillian Community understand this pain, and continuously strive to provide users a seamless
+experience for writing integration tests with ease by bringing their tests to the real environment and
+deploy with confidence like never before.
+
+With this blog post, we aim at showcasing integration tests for your Kubernetes or OpenShift clusters using just "Arquillian Cube":https://github.com/arquillian/arquillian-kube. Moving a step further, we also demonstrate building and deploying from scratch, applications with no deployment configuration, leveraging the power of "Fabric8 Maven Plugin":https://maven.fabric8.io/ along with the Arquillian Kube Extension.
+
+h2. Deployment Testing Recipes
```

Comment:
Maybe we should emphasize on why such tests are useful? I'm missing a clear goal for that.